library/board.py

```diff
@@ -53,7 +65,7 @@ def check_grid_victory(grid: npt.NDArray[np.str_]) -> typing.Optional[str]:
             # .diagonal() gets us a single diagonal list,
             # so these two rows are just True/False whether a diagonal is won
             diagonal_r = ticks.diagonal().sum() == 3
-            diagonal_l = ticks.fliplr().diagonal().sum() == 3
+            diagonal_l = np.fliplr(ticks).diagonal().sum() == 3  # type: ignore
```

Comment:
this `type: ignore` can ignore just a specific type of error so you don't accidentally cause other errors. here's how:
  1. remove the `type: ignore`
  2. run mypy with `--show-error-codes`
  3. copy the error code (looks like `[... stuff here ...]`) into the comment, like: `# type: ignore [... stuff here ...]`

Reply:
Changed this to `# type: ignore [no-untyped-call]`